src/obb/server/http_component.clj

```diff
@@ -0,0 +1,39 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.http-component
+  "The aleph server as a component"
+  (:require [com.stuartsierra.component :as component]
+            [ring.middleware.cors :as cors]
+            [aleph.http :as http]
+            [compojure.core :as compojure]
+            [obb.server.routes :as routes]))
+
+(defn- setup-cors
+  "Setup cors"
+  [handler]
+  (cors/wrap-cors handler
+                  :access-control-allow-origin
+                  [#"^http://localhost(.*)"]
+                  :access-control-allow-methods [:get :put :post :delete]))
+
+(def app
+  "The main app handler"
+  (-> (compojure/routes routes/public-routes)
+      (setup-cors)))
+
+(defrecord HttpServerComponent [server port]
+
+  component/Lifecycle
+
+  (start [component]
+    (println (str "** OBB HTTP Server " (or (get (System/getenv) "OBB_ENV" "development"))
```

Comment:
I'm not worrying about this, still leaving this for later and maybe waiting for more config stuff (like db). But I agree with you, I like that "blow up if no configs are set". However, how do you handle the default dev configs?